UPGRADE.md

```diff
@@ -1,5 +1,65 @@
 # Upgrade
 
+## unreleased
+
+### Sitemap Provider changed
+
+As a sitemap is always domain specific and a domain can have multiple webspaces and portal
+the `SitemapProviderInterface` changed. To make it possible to provide also none Sulu
+routes the `SitemapUrl` constructor changed and now needs as third paramter a defaultLocale.
+
+```php
+// before
+public function build($page, $portal) {
+     return [
+          new SitemapUrl('/test-1', 'de');
+     ];
+}
+
+public function createSitemap() {
+    return new Sitemap($alias, $this->getMaxPage()/*, $lastMod */);
+}
+
+public function getMaxPage() {
+     return 1;
+}
+
+// after
+public function build($page, $scheme, $host) {
+     return [
+          new SitemapUrl('http://test.lo/test-1', 'de', 'de');
+     ];
+}
+
+public function createSitemap() {
+    return new Sitemap($this->getAlias(), $this->getMaxPage()/*, $lastMod */);
+}
+
+public function getAlias(): string {
+    return 'myalias';
+}
+
+public function getMaxPage($scheme, $host) {
+     return 1;
+}
+```
+
+In this case also the `XmlSitemapDumper` and `XmlSitemapRender` does not longer need PortalInformations.
```

Comment:
"In this case" does not really fit here... How about "The `XmlSitemapDumper` and `XmlSitemapRenderer` no longer need `PortalInformations`."